conf/services/penalties-appeals

```diff
@@ -0,0 +1,27 @@
+serviceName: VAT Penalties and Appeals Reform
```

Comment:
The IT's fail due to the file not having an extension, e.g. .yml

Reply:
Fixed, along with date positioning